src/networks.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
   LOCAL: 'local',
   EVONET: 'evonet',
   TESTNET: 'testnet',
+  MAINNET: 'mainnet',
```

Comment:
I think we can keep it because instead of removing all the conditional checks for testnet and adding them again later as we prepare for mainnet, we can just switch them to conditional checks for mainnet now. It is not possible to use the `mn setup` wizard for mainnet, it will respond with the error `Config with name 'mainnet' is not present` because there is no template yet.

Reply:
Got it!